src/wallet/scriptpubkeyman.h

```diff
@@ -154,6 +154,9 @@ class ScriptPubKeyMan
     virtual void KeepDestination(int64_t index) {}
     virtual void ReturnDestination(int64_t index, bool internal, const CPubKey& pubkey) {}
 
+    /** Fills internal address pool. Should be used sparingly within ScriptPubKeyMan implementations.
```

Comment:
This is a vaguely unactionable comment.

Can we give a positive statement about when it's to be used? The last two uses appear to be when the old keypool is being blown away.

Reply:
Expanded the comment a bit more.

It's kind of hard to say when it should and should not be used. I'm just trying to avoid the previous behavior of sprinkling TopUps everywhere.

Reply:
fair enough.